events/2023-google-summer-of-code/index.markdown

```diff
@@ -41,16 +32,7 @@ For general MoveIt GSoC questions you can contact [Mark Moll](mailto:mark@pickni
 * Preferred skills: C++, Functional Programming, and GTest
 * Mentor: [Tyler Weaver](mailto:tyler@picknik.ai), [Michael Görner](mailto:goerner@informatik.uni-hamburg.de)
 
-### Bullet collision checking integration
-
-* Bullet support is partially done, but more work is needed. [A memory leak issue](https://github.com/ros-planning/moveit_task_constructor/issues/232#issuecomment-774249206) was identified that needs to be fixed. Support for continuous collision checking would be nice. Related pull requests include [PR 2897](https://github.com/ros-planning/moveit/pull/2897) and [PR 2838](https://github.com/ros-planning/moveit/pull/2838).
-* Project size: medium (175 hours)
-* Expected outcome: Continuous collision checking is supported and the memory leak is eliminated.
-* Difficulty: hard
-* Preferred skills: MoveIt, Collision Checking, and C++
-* Mentor: [Simon Schmeisser](mailto:simon.schmeisser@optonic.com), [Michael Görner](mailto:goerner@informatik.uni-hamburg.de)
-
-### Better Parallel Planning with MoveIt 2
+### Better Parallel Planning [MoveIt 2]
```

Comment:
Doesn't this target [MoveIt 1 just as well](https://github.com/ros-planning/moveit/pull/3244)?

Also I still think a lot of the features read like "replicate what MTC already does", so maybe it's actually an option to target a parallel container implementation of this in MTC. :clown_face: